### ISSUE TITLE
WhatApps works with Google Chrome 60+

### ISSUE THREAD

**Issue by gianmarcoalberti** [meetfranz/franz#2116]
**Describe the bug**
After starting FRANZ, no service is loaded and the message "WhatApps works with Google Chrome 60+" pops up. Everything was working fine till 10 Dec 2021. FRANZ version 5.4.1

**To Reproduce**

**Expected behavior**
I would expect that the services could be loaded.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mojave 10.14.6

**Smartphone (please complete the following information):**

**Additional context**
I was using FRANZ using just one service (Whatsapp), which I could use anymore due to the reported issue. I tried to test the behaviour by adding a new service (i.e., another Whatsapp account). It worked for few seconds, then it started not working as the other Whatsapp service.

**Comment by alansaviolobo** [comment]
I second this.
I initially had version 5.4 and I also tried updating to version 5.7 but with no luck.

The thing started when there was a message stating that some whatsapp files had updated and that franz/whatsapp tab needed to reload

**Comment by gianmarcoalberti** [comment]
@alansaviolobo yes...you are correct...as far as I recall, at the bottom of the FRANZ window there was a message saying something like that. I did not pay attention tough. I closed FRANZ, and by the time I opened it again, it started not working.

**Comment by nicolabisio** [comment]
@gianmarcoalberti I've fixed the issue by deleting service and create it again. 
I hope this help

**Comment by gianmarcoalberti** [comment]
@nicolabisio Thanks for the suggesiton. I gave it a try, but when I close and start FRANZ again the same issue crops out.

**Comment by alansaviolobo** [comment]
> @gianmarcoalberti I've fixed the issue by deleting service and create it again. I hope this help

@nicolabisio  yes this appears to have resolved the issue. :+1:

**Comment by GleitonCampos** [comment]
Remove the service and add it again to resolve the issue

**Comment by shooby83** [comment]
> Remove the service and add it again to resolve the issue

Could you give further informations on how to do that?

**Comment by nicolabisio** [comment]
> > Remove the service and add it again to resolve the issue
> 
> Could you give further informations on how to do that?

I've removed the service from Settings > Your Services
![image](https://user-images.githubusercontent.com/2479973/145583245-a10bfdd4-7eee-4caa-b38c-7bd438ac006c.png)
![image](https://user-images.githubusercontent.com/2479973/145584232-48ab70b5-99a1-485f-b476-5bed64d18261.png)

Then create it again

**Comment by shooby83** [comment]
> I've removed the service from Settings > Your Services

Oh god, thank you :-D . I thought you guys meant some sort of system services in your OS.

Thanks for the clarification. And YES! It worked!

**Comment by mostrosonido** [comment]
> @gianmarcoalberti I've fixed the issue by deleting service and create it again. I hope this help

It works for me +1

**Comment by adri2906** [comment]
> > I've removed the service from Settings > Your Services
> 
> Oh god, thank you :-D . I thought you guys meant some sort of system services in your OS.
> 
> Thanks for the clarification. And YES! It worked!

It reappears when FRANZ is relaunched, the problem lies within Whatsapp web and the FRANZ container as for a couple of years ago...

**Comment by gianmarcoalberti** [comment]
I did manage to fix the issue, which crops out (as noted) after re-launching FRANZ. I have temporarily switched to Ferdi, which is working so far.
